pkg/apis/security/v1alpha1/types.go

```diff
@@ -185,3 +194,36 @@ type ClusterNetworkPolicyList struct {
 
 	Items []ClusterNetworkPolicy `json:"items"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type Tier struct {
```

Comment:
Does it make sense to call it NetworkPolicyTier?

Reply:
i think @McCodeman came up with Tier name.. i am not against new name suggestion